packages/backend.ai-ui/src/components/fragments/BAIImportArtifactModal.tsx

```diff
@@ -164,14 +164,16 @@ const BAIImportArtifactModal = ({
               );
               onOk(
                 e,
-                res.importArtifacts.tasks.map((task) => ({
-                  taskId: task.taskId,
-                  version: task.artifactRevision.version,
-                  artifact: {
-                    id: toLocalId(selectedArtifact?.id ?? ''),
-                    name: selectedArtifact?.name ?? '',
-                  },
-                })),
+                res.importArtifacts.tasks
+                  .filter((task) => task.taskId != null)
+                  .map((task) => ({
+                    taskId: task.taskId!,
+                    version: task.artifactRevision.version,
+                    artifact: {
+                      id: toLocalId(selectedArtifact?.id ?? ''),
+                      name: selectedArtifact?.name ?? '',
+                    },
+                  })),
```

Comment:
Please check this error:  
![image.png](https://app.graphite.com/user-attachments/assets/187b61f1-4910-429a-865d-1cee2f01783a.png)